asmcli/commands/create-mesh.sh

```diff
@@ -154,3 +154,15 @@ prepare_add_to_mesh_environment() {
     organize_kpt_files
   fi
 }
+
+generate_secret_name() {
+  local SECRET_NAME; SECRET_NAME="${1}"
+
+  if [[ "${#SECRET_NAME}" -gt 63 ]]; then
```

Comment:
very nit: since we do this same check somewhere else, maybe we should pull KUBE_TAG_MAX_LEN into a readonly variable in util.sh?

Reply:
Done